app/models/proxy_rule.rb

```diff
@@ -25,6 +28,16 @@ class ProxyRule < ApplicationRecord
 
   ALLOWED_HTTP_METHODS = %w[GET POST DELETE PUT PATCH HEAD OPTIONS].freeze
 
+  scope :by_query, ->(query, owner_type, owner_id) do
```

Comment:
RuboCop is complaining about this right now, but just rebase from master and it won't complain about this anymore :wink: